src/axom/sidre/core/Group.hpp

```diff
@@ -1662,6 +1662,22 @@ class Group
    */
   bool loadExternalData(const hid_t& h5_id);
 
+  /*!
+   * \brief Load data into the Group's external views from a path into
+   *        hdf5 handle
+   *
+   * No protocol argument is needed, as this only is used with the sidre_hdf5
+   * protocol.  Returns true (success) if no Conduit I/O error occurred since
+   * this Group's DataStore was created or had its error flag cleared; false,
+   * if an error occurred at some point.
+   *
+   * \param h5_id        hdf5 handle
+   * \param group_path   path to a group below the group pointed by the h5_id.
```

Comment:
```suggestion
   * \param group_path   path to a group below the group pointed to by the h5_id.
```

I find "group below the group pointed to" a bit confusing.

Reply:
I'm trying to reword it.  There's a group inside the HDF5 file that corresponds to the `this` Group in the Sidre hierarchy, and we need to pass the path to find it in the file.  That becomes confusing to explain succinctly.

Reply:
> Perhaps "HDF5 path from h5_id to the target group" ?

That isn't quite accurate, because in the `sidre_hdf5` protocol the actual path from `h5_id` to the group is `"sidre/external/" + group_path`, but that's an implementation detail, not something exposed in the API.

The path needed here is the path from the top Group that was written to the file to `this` Group which we are trying to load with data.  It's hard to write that in a way that doesn't become confusing with "group" used multiple times, and the word "group" can mean either a Sidre Group or an HDF5 group.